src/libraries/Common/tests/System/Net/Http/Http3LoopbackConnection.cs

```diff
@@ -195,6 +190,17 @@ public async Task EstablishControlStreamAsync(SettingsEntry[] settingsEntries)
             await _outboundControlStream.SendSettingsFrameAsync(settingsEntries);
         }
 
+        public async Task DisposeCurrentStream()
+        {
+            Assert.NotNull(_currentStream);
+            Assert.True(_currentStreamId >= 0);
+
+            await _currentStream.DisposeAsync().ConfigureAwait(false);
+            _openStreams.Remove((int)_currentStreamId);
```

Comment:
This breaks the numeration alignment that seemed to be present before (`_openStreams[requestId - 1]`) -- is that ok? we are not using it anywhere anymore?

Reply:
Ah, that's a dictionary, not a list 🤦‍♀️ sorry 😄